docs/marketplace/user/merchant-portal-user-guides/202108.0/offers/managing-product-offers.md

```diff
@@ -83,41 +69,59 @@ You can sort the existing product offers list by:
 - Valid from date
 - Valid to date
 
-##### Searching product offers
+#### Searching product offers
 
-You can search the existing offers by:
+Using the search field in the top right corner, you can search product offers by:
 
 - Offer reference
 - merchant reference
 - product SKU
 
-### Create offer page
+### Create Offer page
+
+On the *Create Offer* page, you see the list of product you can create an offer for. You can filter, sort and search the products.
+
+#### Filtering products
+
+Using the filtering menu at the top of the page, you can filter products by:
+
+- Existing offers
+- Visibility
+
```

Comment:
Please specify the field names where you can filter offers by existing and visibility. It's not obvious, as the fields for filtering are called just "offers" and "status"